bsdlib/doc/at_commands.rst

```diff
@@ -0,0 +1,58 @@
+.. _at_commands:
+
+Sending AT commands
+###################
+
+You can use the BSD socket interface to send AT commands to the LTE modem and to receive the responses.
+
+To do so, create a socket with the proprietary address family NRF_AF_LTE and the protocol family NRF_PROTO_AT.
+You can then write AT command strings to that socket, and they are sent to the modem.
```

Comment:
`[..] to that socket using nrf_send() and nrf_write()` ?

Reply:
Do you have an answer to my question about what is the difference between those two? https://github.com/NordicPlayground/nrfxlib/pull/121#discussion_r357629694

Reply:
The only difference `send()` and `write()` is the presence of _flags_.
With a zero _flags_ argument, `send()` is equivalent to `write()`.